webserver/Startup.cs

```diff
@@ -3,6 +3,7 @@
 using Microsoft.AspNetCore.Identity;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.OpenApi.Models;
```

Comment:
Do you need this import in the Startup class?
It's seems that is not being used.